src/styles/variables.scss

```diff
@@ -0,0 +1,4 @@
+$main-color: #060b35;
+$secondary-color: #616070;
+$font-family: "Roboto", sans-serif;
```

Comment:
https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md
2. [STYLES] - Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)